src/main/kotlin/blackjack/controller/BlackjackController.kt

```diff
@@ -13,21 +14,27 @@ import blackjack.view.OutputView
 class BlackjackController {
     fun start() {
         val deck = CardDeckGenerator.generate()
-        val participants = initializeGameSetting(deck, InputView.readNames())
+        val playerNames = InputView.readNames().map { ParticipantName(it) }
+        val participants = initializeGameSetting(deck, playerNames)
         OutputView.printInitialStatus(participants)
 
         playRound(deck, participants)
         OutputView.printStatusAndScore(participants)
-        OutputView.printResult(participants.getDealerWinningState(), participants.getPlayerWinningState())
+
+        val playerProfits = participants.getPlayerProfits()
+        val dealerProfits = participants.getDealerProfits(playerProfits)
+        OutputView.printProfit(dealerProfits, playerProfits)
     }
 
     private fun initializeGameSetting(
         deck: CardDeck,
-        names: List<String>,
+        playerNames: List<ParticipantName>,
     ): Participants {
+        val players =
+            playerNames.map { playerName ->
+                Player(playerName, createInitialHand(deck), BetAmount(InputView.readBetAmount(playerName)))
```

Comment:
```suggestion
                Player(playerName, createInitialHand(deck), BetAmount(amount))
```
호출은 최대한 중첩하지 않도록 신경써보면 좋을 것 같아요.
중첩이 두번이상 생길 수록 가독성이 급격하게 떨어질 것 같습니다.

Reply:
넵! 해당 부분 분리하겠습니다~